javascript/app.js

```diff
@@ -22,3 +23,194 @@ function closeMenuHandler() {
 menuBars.addEventListener('click', openMenuHandler);
 closeMenuBtn.addEventListener('click', closeMenuHandler);
 mobileLinks.addEventListener('click', closeMenuHandler);
+
+const worksArray = [
+  {
+    id: 0,
+    title: 'Tonic',
+    experiences: ['Canopy', 'Back End Dev', '2015'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    urlMobile: '/images/work_1.jpg',
+    urlDesktop: '/images/work_1_lg.png',
+    urlModal: '/images/work_modal.png',
+    alt: 'work_tonic_daily_personalized_reads',
+    live: 'https://dennis-every.github.io/',
+    source: 'https://github.com/dennis-every/portfolio',
+  },
```

Comment:
- 👏🏻 👏🏻 Excellent work implements an array of objects to store JavaScript objects containing the information for each project. 😉  💯